src/Symfony/Component/Cache/Traits/AbstractTrait.php

```diff
@@ -159,6 +169,27 @@ public function deleteItems(array $keys)
     }
 
     /**
+     * Enables/disables versioning of items.
+     *
+     * When versioning is enabled, clearing the cache is atomic and doesn't require listing existing keys to proceed,
+     * but old keys may need garbage collection and extra round-trips to the back-end are required.
+     *
+     * Calling this method also clears the memoized namespace version and thus forces a resynchonization of it.
```

Comment:
@nicolas-grekas Spelling error "memoized" -> "memorized"

Reply:
I reported the "memoized" typo once ... and I was wrong :(  "Memoize" seems to be a valid concept: https://en.wikipedia.org/wiki/Memoization  Maybe it's used correctly this time too?

Reply:
Ah; seems you are correct! I think the latter one is a spelling error though. ;-)

Reply:
I wrote memoize on purpose :)